src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/PosixSignalRegistrationTests.Unix.cs

```diff
@@ -57,7 +59,15 @@ public void SignalHandlerCalledForKnownSignals(PosixSignal s)
                     semaphore.Release();
                 });
 
-                kill(signal);
+                // Use 'kill' executable with signal name to validate the signal pal mapping.
+                using var process = Process.Start(new ProcessStartInfo
+                    {
+                        FileName = "kill",
+                        ArgumentList = { "-s", signalStr, Environment.ProcessId.ToString(CultureInfo.InvariantCulture) }
```

Comment:
The invariant culture shouldn't be needed.

Reply:
Is something setup specifically for tests?

Reply:
No, but culture only matters for negative values in order to determine what negative sign to use.